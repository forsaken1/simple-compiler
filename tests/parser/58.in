int f(int i, int j) { return 0; }