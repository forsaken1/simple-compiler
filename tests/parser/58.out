int function (f)
|
+---(,)
|   |
|   +---(j)
|   |
|   +---(i)
|
+---{stmt}
    |
    +---(return)
        |
        +---(0)